.ci/Jenkinsfile

```diff
@@ -12,6 +12,11 @@ pipeline {
     PIPELINE_LOG_LEVEL='INFO'
     JOB_GCS_BUCKET = 'fleet-ci-artifacts'
     JOB_GCS_CREDENTIALS = 'fleet-ci-gcs-plugin-file-credentials'
+
+    DOCKER_REGISTRY = 'docker.elastic.co'
+    DOCKER_REGISTRY_SECRET = 'secret/observability-team/ci/docker-registry/prod'
+
+    DOCKER_IMAGE = "${env.DOCKER_REGISTRY}/fleet-server/fleet-server"
```

Comment:
This is the docker image that will be pushed @joshdover 

AFAIK using that would be a public docker image. How could we set it private ?

In this comment was mentioned to not use employees path 
https://github.com/elastic/fleet-server/issues/2352#issuecomment-1455872594

Reply:
In other repos I see that for instance it is used `docker.elastic.co/observability-ci/*` , are they private? 

For instance:
https://github.com/elastic/package-registry/blob/70faae3990e811ae5c706e6b6caac7288b777a75/.ci/Jenkinsfile#L14

Reply:
Using for now docker.elastic.co/observability-ci/fleet-server as docker image